### ISSUE TITLE
Fix - Color Contrast Ratio Fails Accessibility Test

### ISSUE THREAD

**Issue by sieis** [sieis/unmove#17]
The color of <a> links fails the accessbility test. Replace with acceptable blue.

Test in Question:

![image](https://user-images.githubusercontent.com/3012159/186443208-02724ab0-10e8-4420-8c69-dd7e54c1a7b2.png)

Suggested Color to Fix using [WebAIM Color Tool](https://webaim.org/resources/contrastchecker/):

**#0000FF**

![image](https://user-images.githubusercontent.com/3012159/186443422-ae5022bc-cacf-48e3-b8f3-df9e9f1b5889.png)
